docs/sdks/survey-sdk/realtime-responses-import.md

```diff
@@ -0,0 +1,87 @@
+# Realtime Responses Import
+
+As explained in [Getting Started](../quick-start) you can enable the import of responses in realtime, by doing so you'll obtain a **Realtime endpoint URL** which you'll be able to use to perform HTTP POST requests in order to import survey responses in realtime (see [Payload example](#payload-example) for more detail on the payload expected by Engage Digital).
```

Comment:
> You'll obtain a **Realtime endpoint URL**

You don't explain where they'll obtain it, I find this a bit confusing WDYT ?

Reply:
I rephrased this whole part in [a1fcef2](https://github.com/ringcentral/engage-digital-api-docs/pull/80/commits/a1fcef26c49b65d7cd05340e638c1331a1221772), let me know if it's clearer this way

Reply:
Perfect ! Even I wasn't sure it was in the survey config page either 👍